packages/apps/docs/src/hooks/useSearch/useStream.ts

```diff
@@ -73,15 +75,16 @@ export const useStream = (): [
         try {
           const data = JSON.parse(event.data);
           setMetadata(data);
-        } catch (error) {
-          //@TODO: add error message in UI
-          console.error(error);
+        } catch (e) {
+          console.warn('metadataListener', e);
+          setError(e.message);
+          done();
         }
       });
 
-      source.addEventListener('error', (error) => {
-        //@TODO: add error message in UI
-        console.log(error);
+      source.addEventListener('error', (e) => {
+        console.warn('sourceError', e);
+        setError('Error in the source event listener');
```

Comment:
maybe a bit more user friendly error messages
lets go for `There was a problem. Please try again at a later time. Sorry for the inconvience`

We can use this error for all at the moment.
Else maybe check with Isa what a good error message could be.

Reply:
I like it. But maybe we shouldn't send away the user to try again later but instead try again.

`There was a problem. Sorry for the inconvenience. Please try again!`